Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt

```diff
@@ -37,6 +38,12 @@ class AndroidModule {
         @AppContext context: Context
     ): NotificationManagerCompat = NotificationManagerCompat.from(context)
 
+    @Provides
+    @Singleton
+    fun notificationManager(
+        @AppContext context: Context
+    ): NotificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
```

Comment:
There is a handy extension for that in `core-ktx`
`context.getSystemService< NotificationManager>`

Reply:
I preferred the original solution without the `!!`

Reply:
I understand, but it has the same result at the end. `getSystemService` returns a nullable `NotificationManager`casting it to a NonNullable type would throw a null exception pointer too.